code/admin/tables/log.php

```diff
@@ -135,6 +144,8 @@ public function store($updateNulls = false)
 			$this->post_data = JArrayHelper::toString($this->post_data, '=', '&');
 		}
 
+		$this->request_method = JFactory::getApplication()->input->getMethod();
```

Comment:
this shall be part of past_data @niteshkesarkar

Reply:
@manojLondhe - Updated accordingly.